src/main/java/athena/exceptions/AddException.java

```diff
@@ -0,0 +1,14 @@
+package athena.exceptions;
+
+import athena.Ui;
+
+public class AddException extends Exception{
+    public AddException() {
+
+    }
+
+    public void getErrorMessage() {
```

Comment:
`getErrorMessage` gives an impression that it is returning something. Perhaps `printErrorMessage` instead? Or return the error message as a string from this?

Reply:
Also remember to add Javadoc comments for this